packages/opentelemetry/lib/index.js

```diff
@@ -0,0 +1,89 @@
+const { diag, DiagLogLevel } = require('@opentelemetry/api');
+const {
+	getNodeAutoInstrumentations
+} = require('@opentelemetry/auto-instrumentations-node');
+const { Resource } = require('@opentelemetry/resources');
+const opentelemetrySDK = require('@opentelemetry/sdk-node');
+const {
+	SemanticResourceAttributes
+} = require('@opentelemetry/semantic-conventions');
+const appInfo = require('@dotcom-reliability-kit/app-info');
+const {
+	OTLPTraceExporter
+} = require('@opentelemetry/exporter-trace-otlp-proto');
+const { NoopSpanProcessor } = require('@opentelemetry/sdk-trace-base');
+const logger = require('@dotcom-reliability-kit/logger');
+// OpenTelemetry Traces Endpoint to which to export traces (e.g. http:localhost:4318/v1/traces).
+/**
+ * Set up OpenTelemetry tracing.
+ *
+ * @param {object} options
+ *      Options passed into the function parameter.
+ * @param {string | null} [options.tracesEndpoint]
+ *      The URL to send OpenTelemetry trace segments to, for example http://localhost:4318/v1/traces.
+ */
+function opentelemetry({ tracesEndpoint }) {
```

Comment:
**nitpick:** I think maybe we could make this method more descriptive of what it's doing, maybe something like `setupOpenTelemetry`?

Reply:
Yeah good point